recipes/ua-nodeset/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.02-2021-07-21":
```

Comment:
Reading the releases, they have different prefixes, but are related to the same project. How can distribute it individually? Here you are using PADIM, but what does happen if someone want DEXPI? It looks the project is the same, but the provided package is totally different.

Or you could use the real release name for package version, or we should think about separated packages for each release, like `ua-nodeset-padim/1.02`

Reply:
These tags are like rolling releases on the 1.04 branch.
E.g. the tag `PADIM-1.02-2021-07-21` (https://github.com/OPCFoundation/UA-Nodeset/releases/tag/PADIM-1.02-2021-07-21) is for the commit https://github.com/OPCFoundation/UA-Nodeset/commit/6f2b38413fc401cb0837954fdb90c6ecb3ceffeb which is almost the newest one on the 1.04 branch.

The tag names just indicate that a specific companion specification version is part of that release. All other companion specifications released previously are still part of that too.

Therefore I do not think that it is a good idea to separate the ua-nodeset package into sub-packages. 

I suggest to just use the version `1.04` and the date of the latest tag, i.e., `1.04-2021-09-15`.
I will update the conandata.yml file with some more comments (and also fix the version as it is currently wrong)